app/templates/direct-award/did-you-award-contract.html

```diff
@@ -0,0 +1,70 @@
+{% extends "toolkit/layouts/_base_page.html" %}
+
+{% block page_title %}Did you award a contract? - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+        "link": url_for('main.index'),
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for('external.buyer_dashboard'),
+        "label": "Your account"
+      },
+      {
+        "link": url_for('direct_award.saved_search_overview', framework_framework=framework.framework),
+        "label": "Your saved searches"
+      },
+      {
+        "link": url_for('direct_award.view_project', framework_framework=framework.framework, project_id=project.id),
+        "label": project.name
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="did-you-award-contract-page">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header class="page-heading">
+        <h1>
+          Did you award a contract for '{{project.name}}'?
+        </h1>
+      </header>
+
+      <form method="POST" action="{{ url_for('direct_award.did_you_award_contract', framework_framework=framework.framework, project_id=project.id) }}">
```

Comment:
It may be possible to use `action=""` and if so that would probably be preferable?

Reply:
According to [SO](https://stackoverflow.com/questions/1131781/is-it-a-good-practice-to-use-an-empty-url-for-a-html-forms-action-attribute-a) `action=""` _can_ cause issues.

I agree it is nicer though not to have a url in there. Is there any reason not just to leave the `action` attribute out altogether? Thoughts @DilwoarH?

Reply:
https://stackoverflow.com/a/9401608

Have a read of the above, I personally would leave it as it is as there may be some browsers which behave differently without an action. It's also standard to have an action target - removing it won't bring any additional benefit.

Reply:
Okay, I've left it as it is, but also added a test `test_award_contract_form_action_url_is_award_contract_url` that checks that the `action` url is the same as the GET url